packages/manager/apps/hpc-vmware-managed-vcd/src/data/hooks/useUpdateVcdDatacentre.ts

```diff
@@ -0,0 +1,52 @@
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { useQueryClient, useMutation } from '@tanstack/react-query';
+import { useState } from 'react';
+import {
+  UpdateVdcDetailsParams,
+  updateVdcDetails,
+} from '../api/hpc-vmware-managed-vcd-datacentre';
+
+const updateVdcDetailsQueryKey = ({
+  id,
+  vdcId,
+}: Pick<UpdateVdcDetailsParams, 'id' | 'vdcId'>) => [
+  `put/vmwareCloudDirector/organization/${id}/virtualDataCenter/${vdcId}`,
+];
+
+export const useUpdateVdcDetails = ({
+  id,
+  vdcId,
+  onSuccess,
+  onError,
+}: {
+  id: string;
+  vdcId: string;
+  onSuccess?: () => void;
+  onError?: (result: ApiError) => void;
+}) => {
+  const [isErrorVisible, setIsErrorVisible] = useState(false);
```

Comment:
Why don't you use `isError` from the hook useMutation instead ?

Reply:
I thought that was by convention the way it's handled, as in key-management-services & vrack-services and I did not want to follow different patterns from the already present ones